### PR TITLE
2. As a user, I want to be able to navigate within the app

### PR DESCRIPTION
**Pull request by mikeramz86**
✅ react-router-dom has been added as a project dependency

✅ Click a link within the app to show a different view in the app

- We looked at our old projects to figure out how to create a new component

✅ When a link is clicked, the browser URL updates to represent the current view

- At first, we manage to create a link that appeared on the main screen however unsuccessful to create a new screen
- Nikema, then took time to review more of her notes and researched a better a solution
- She created a new commit and we synced to go over the new code lines

**Questions:**

- Nikema: How to set up the Router so that the Home Page is the App.js and have the link to switch to a new view?
- Is there a better solution/more efficient solution for this?
- Mike: Do we need `switch` or `react-router-dom`?

### CODE REVIEW

**Comment by larsbrekken** [comment]
I see that at some point you renamed the hello.js file, but looks like maybe that rename didn't fully make it into git? Did you miss a `git add` or similar?

image

As I was typing this I just realized that git is not especially great at renaming files if all you are changing is the casing (e.g. from `hello` to `Hello`).

One solution here may be to change the file name entirely, maybe give that a shot?

**Comment by larsbrekken** [comment]
Note the lowercase `hello` vs the uppcase `Hello`

File: src/AppRouter.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import Hello from './hello';
+import Hello from './Hello';
```
Comment (stevelikesmusic) [inline src/AppRouter.js]: Nice catch with the name capitalization. Like @larsbrekken mentioned, we'll need to change the name of the file as well now from `src/hello.js` to `src/Hello.js`

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,24 @@
+// import React from 'react';
```
Comment (stevelikesmusic) [inline src/index.js]: This can all get removed 🔥

File: src/HelloAgain.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function HelloAgain() {
+  return (
+    <div>
```
Comment (stacietaylorcima) [inline src/HelloAgain.js]: Not a blocker by any means, but for better accessibility, you might consider using semantic HTML here as a `div` essentially has no meaning. I might suggest using `main` here in this case. You also could use a React Fragment if you ever come across a case where you need to wrap elements for some reason but they don't really fit with any of the semantic tags.

**Pull request merged**